### ISSUE TITLE
Player view don't work while watching the game

### ISSUE THREAD

**Issue by ajaykauthale1** [ValveSoftware/Dota2-Gameplay#11050]
### Description

When watching matches if I select player view rather than directed camera, then it no longer works.

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/22283064/f4c40a0e-7ed0-4765-ab01-307b99cc71ca)

**Comment by Avarise831** [comment]
#11026

**Comment by diiman2000** [comment]
This is true. tested and the player view does not work...BUT if you want a quick fix u can use the hero chase option. It is the new player view..seems there is a mixup but hero chase functions as player view now. Try and let me know.

**Comment by muciflex** [comment]
> This is true. tested and the player view does not work...BUT if you want a quick fix u can use the hero chase option. It is the new player view..seems there is a mixup but hero chase functions as player view now. Try and let me know.

That is not good , I want to see the player when he is looking at the map or other parts , where he is moving his mouse . On Hero Chase you do not see that .

**Comment by ajaykauthale1** [comment]
@robert-gogolan yes it is working now. Please close this issue.
